<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>11</y>
      <width>241</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>ADC 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_ADC1">
           <property name="text">
            <string>0.000 V.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>ADC 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_ADC2">
           <property name="text">
            <string>0.000 V.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>BH1750</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="L_BH1750">
           <property name="text">
            <string>00.00 lx.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget111">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>221</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="L_DK1">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>72</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;DK1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_DK2">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>72</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;DK2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="L_MOTION">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>72</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MOTION&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="RB_Update">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget212">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>233</width>
      <height>148</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>PWM1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="S_PWM1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>PWM2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="S_PWM2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>PWM3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="S_PWM3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>ALL PWM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="S_ALLPWM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>172</width>
           <height>16</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="B_Connect">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget131313">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>154</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="CustomLabel" name="L_LED1">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LED1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LED1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="CustomLabel" name="L_LED2">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LED2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LED2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="CustomLabel" name="L_REL1">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;REL1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;REL1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="CustomLabel" name="L_REL2">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;REL2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;REL2&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="RB_AutomaticControl">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatic Control</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_SerialPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Scan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_ID">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2047</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_Log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>581</width>
      <height>191</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Times New Roman';&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAutomatic_control_setting"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionAutomatic_control_setting">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Automatic control setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header location="global">CustomLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
